openapi_tester/schema_tester.py

```diff
@@ -513,16 +513,6 @@ def test_openapi_array(
         ignore_case: Optional[List[str]],
     ) -> None:
         items = schema_section["items"]  # the items keyword is required in arrays
-        if data is None:
-            if self.is_nullable(schema_section):
-                return
-            raise DocumentationError(
-                message=NONE_ERROR.format(expected="list"),
-                response=data,
-                schema=schema_section,
-                reference=f"{reference}.array",
-                hint="Document the contents of the empty dictionary to match the response object.",
-            )
```

Comment:
Are we handling this somewhere else?

Reply:
Yup, but we should do this just once,not twice